### PR TITLE
chore: Fix GH security warnings

### PR DESCRIPTION
**Pull request by maraisr**
Albeit low impact; there are security warnings I'd like to see disappear. And I mean, what even is yalc 🤷‍♂ 

image

Re yalc; as a new-comer to the project, that clones the example to have a play, they wouldn't have yalc, nor can we guarantee that they have built the language compiler. Should we maybe pin a version, or at the very least a tag to install? And for us that use that as a playground when trying new things, we'll we could locally override that, or fallback to yalc if we choose?

### CODE REVIEW

File: example-hooks/package.json [hunk 1]
```diff
@@ -48,11 +48,11 @@
     "babel-preset-stage-0": "^6.5.0",
     "babel-runtime": "^6.26.0",
     "csstype": "^2.5.6",
-    "fork-ts-checker-webpack-plugin": "^0.3.0",
+    "fork-ts-checker-webpack-plugin": "^3.1.1",
     "patch-package": "^6.2.0",
     "postinstall-postinstall": "^2.0.0",
     "relay-compiler": "^7.0.0",
-    "relay-compiler-language-typescript": "file:.yalc/relay-compiler-language-typescript",
+    "relay-compiler-language-typescript": "file:../lib",
```
Comment (alloy) [inline example-hooks/package.json]: Why is this being included in this PR?

File: example/package.json [hunk 2]
```diff
@@ -48,9 +48,9 @@
     "babel-cli": "^6.26.0",
     "babel-eslint": "6.1.2",
     "csstype": "^2.5.6",
-    "fork-ts-checker-webpack-plugin": "^0.3.0",
+    "fork-ts-checker-webpack-plugin": "^3.1.1",
     "relay-compiler": "^7.0.0",
-    "relay-compiler-language-typescript": "file:.yalc/relay-compiler-language-typescript",
+    "relay-compiler-language-typescript": "file:../lib",
```
Comment (alloy) [inline example/package.json]: Ditto.

File: example-hooks/package.json (hunk 1 above)
Comment (maraisr) [reply example-hooks/package.json]: My bad... was just trying to make sure things still worked, and not using yalc, this was the fix that allowed that to happen. Must have snuck in.

Did you want me to remove this from the PR?

Comment (alloy) [reply example-hooks/package.json]: Yes please 🙏

**Comment by alloy** [comment]
Bump

**Comment by maraisr** [comment]
Sorry for the delay @alloy  - this is done :)

**Comment by alloy** [comment]
No problemo :) Thanks!

**Comment by alloy** [comment]
:rocket: PR was released in v10.1.5 :rocket:

**Pull request merged**